rotkehlchen/externalapis/monerium.py

```diff
@@ -103,19 +121,24 @@ def get_and_process_orders(self, tx_hash: EVMTxHash | None = None) -> None:
                 (DBCacheStatic.LAST_MONERIUM_QUERY_TS.value, str(ts_now())),
             )
 
-        orders = self._query(
-            endpoint='orders',
-            params={'txHash': tx_hash.hex()} if tx_hash is not None else None,
-        )
+        try:
+            response = self._query(
+                endpoint='orders',
+                params={'txHash': tx_hash.hex()} if tx_hash is not None else None,
```

Comment:
```suggestion
                params={'txHash': str(tx_hash)} if tx_hash is not None else None,
```
you'll save the planet by doing this.